File: javascript/src/models/destinationStatusIn.ts
// this file is @generated

export enum DestinationStatusIn {
  Enabled = "enabled",
  Disabled = "disabled",
}

export const DestinationStatusInSerializer = {
  _fromJsonObject(object: any): DestinationStatusIn {
    return object;
  },

  _toJsonObject(self: DestinationStatusIn): any {
    return self;
  },
};

File: rust/src/models/destination_status_in.rs
// this file is @generated
use std::fmt;

use serde::{Deserialize, Serialize};

#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
pub enum DestinationStatusIn {
    #[serde(rename = "enabled")]
    Enabled,
    #[serde(rename = "disabled")]
    Disabled,
}

impl fmt::Display for DestinationStatusIn {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        let value = match self {
            Self::Enabled => "enabled",
            Self::Disabled => "disabled",
        };
        f.write_str(value)
    }
}
